blablas/ep39/index.md

```diff
@@ -0,0 +1,63 @@
+---
+date: 2020-05-17
+time: 22h:30min
+duration: "01:44:48"
+title: "Self Taught Developers"
+tags: ["tech", "Self-taught", "Software engineering"]
+isNext: false
+video: "2971727423053675"
+published: true
+featured: false
+url: ""
+podcastUrl: https://www.facebook.com/geeksblabla/videos/2971727423053675/
+---
+
+In this episode of GeeksBlabla, we discuss with sekf-taught developers and how they made it professionally.
```

Comment:
In this episode of GeeksBlabla, we discuss with self-taught developers about their experience, how they get started and how they made it professionally